.mailmap

```diff
@@ -1500,6 +1500,8 @@ tborisova <ts.borisova3@gmail.com> <ts_borisova@abv.bg>
 tnzl <you@example.com>
 tools4origins <tools4origins@gmail.com>
 tttc3 <T.Coxon2@lboro.ac.uk>
+u7122029 <111028268+u7122029@users.noreply.github.com>
+u7122029 <u7122029@anu.edu.au>
```

Comment:
There should not be separate entries for the same person:
https://github.com/sympy/sympy/blob/5e50b55cc3ce0731574397d00628e429e23b5e53/.mailmap#L138-L148

Reply:
Just fixed. Sorry about that, I'm sure you've had to say the same thing to so many other contributors.